transport/tchannel/header.go

```diff
@@ -24,12 +24,33 @@ import (
 	"context"
 	"encoding/binary"
 	"io"
+	"strings"
 
 	"github.com/uber/tchannel-go"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/transport/tchannel/internal"
 )
 
+const (
+	// ErrorCodeHeaderKey is the response header key for the error code.
+	ErrorCodeHeaderKey = "rpc-error-code"
+	// ErrorNameHeaderKey is the response header key for the error name.
+	ErrorNameHeaderKey = "rpc-error-name"
+	// ErrorMessageHeaderKey is the response header key for the error message.
+	ErrorMessageHeaderKey = "rpc-error-message"
```

Comment:
It’s a good idea. Note that we strip $tracing$ prefixed callreq headers so they never appear to the handler, so it would be reasonable to do the same so these headers never appear to a caller.